basic_samples/DataViews/Python3/program.py

```diff
@@ -54,14 +62,16 @@ def suppressError(sdsCall):
 
 def createData(ocsClient):
     import random
-    global namespaceId, startTime
+    global namespaceId, startTime, endTime
 
     doubleType = SdsType(id="doubleType", sdsTypeCode=SdsTypeCode.Double)
     dateTimeType = SdsType(id="dateTimeType", sdsTypeCode=SdsTypeCode.DateTime)
 
     pressureDoubleProperty = SdsTypeProperty(id="pressure", sdsType=doubleType)
-    temperatureDoubleProperty = SdsTypeProperty(id="temperature",
+    temperatureDoubleProperty = SdsTypeProperty(id=fieldToConsolidateTo,
                                                 sdsType=doubleType)
+    ambient_temperatureDoubleProperty = SdsTypeProperty(id=fieldToConsolidate,
```

Comment:
Should this be `ambientTemperatureDoubleProperty` so that we can be consistent on casing?